brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3536: Eye icon not visible in editor fixed

### Description
Page title eye icon to hide and unhide title not working, We replaced `<iframe>` with `<embed>`

Previously when we add Google Map in editor after update and refreshing the page eye icon disappears 

### Screenshots
https://bsf.d.pr/i/axDP1i


### Types of changes
replacing `<iframe>` with `<embed>`

### Test Cases
https://bsf.d.pr/v/oFZ2Hq
TestCases

- Enable Spectra One or Astra theme
- Add a new page git it a title eye icon to hide and unhide should be visible
- Add google map block
-  Save the page and refresh
- Eye icon should be visible

### Checklist:
